next.md

```diff
@@ -0,0 +1,7 @@
+- info.js - make info pages interactive (depends by url, not static, fill data.js) - Estimate: 10h from Dilyara, 4h from Sergey
+- design fixes - 2h
+
+  - split files by directories -- need TZ from Sergey (5h from Sergey)
+  - setup webpack more precizly -- need TZ from Sergey (10h from Sergey)
+
+    - deploy on website -- (20h from Sergey)
```

Comment:
add here: setup linters
because all code has dirty lines, bad formatting, useless comments etc
we will fix it manually, but for experience, i think, we'll add linters

Reply:
ok